lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -122,6 +122,27 @@ def test_rasterized_ordering(fig_test, fig_ref):
     ax_test.plot(x+1, y, "-", c="b", lw=10, rasterized=False, zorder=1.2)
 
 
+@check_figures_equal(tol=5)
```

Comment:
```suggestion
@check_figures_equal(tol=5, extensions=['svg', 'pdf'])
```

Do not check this with Agg as I do not think it matters in that case.